docs/userguide.md

```diff
@@ -260,9 +225,35 @@ To use TiSpark with Hive:
     spark.sql("select * from hive_table a, tispark_table b where a.col1 = b.col1").show // join table across Hive and Tispark
     ```
 
+## Load Spark DataFrame into TiDB using TiDB Connector
+TiSpark natively supports writing data to TiKV using Spark Data Source API. ACID is also guaranteed.
+
+For example:
+
+```scala
+val tidbOptions: Map[String, String] = Map(
+  "tidb.addr" -> "127.0.0.1",
+  "tidb.password" -> "",
+  "tidb.port" -> "4000",
+  "tidb.user" -> "root",
+  "spark.tispark.pd.addresses" -> "127.0.0.1:2379"
+)
+
+val customer = spark.sql("select * from customer limit 100000")
+
+customer.write
+.format("tidb")
+.option("database", "tpch_test")
+.option("table", "cust_test_select")
+.mode("append")
+.save()
+```
+
+See [here](./datasource_api_userguide.md) for more details.
+
 ## Load Spark DataFrame into TiDB using JDBC
 
-TiSpark does not provide a direct way to load data into your TiDB cluster, but you can still do it by using JDBC.
+Although TiSpark provides a direct way to load data into your TiDB cluster, you can still do it by using JDBC.
```

Comment:
It is not convincing to use transition here... How about:
```suggestion
While TiSpark provides a direct way to load data into your TiDB cluster, you can also do it by using JDBC.
```

Reply:
Please put this section a deprecated tag.

Reply:
how to add a [deprecated] tag?